frontend/test/metabase/scenarios/collections/timelines.cy.spec.js

```diff
@@ -69,7 +69,7 @@ describe("scenarios > collections > timelines", () => {
 
       cy.findByText("Our analytics events");
       cy.findByText("RC1");
-      cy.findByText(/15/);
+      cy.findByText("AM").should("not.exist");
```

Comment:
@ranquild why don't we maintain the date check as well? `cy.contains(15,)` 
This can be for both cases, by doing that we guarantee a check if we came to have a different format.

Reply:
We have date checks for cases where dates are entered in the input https://github.com/metabase/metabase/blob/f6e4c4b2ea9ac29436452c50ed272e938ff8d25c/frontend/test/metabase/scenarios/collections/timelines.cy.spec.js#L35

Reply:
Here were are really just interested that the event has been created

Reply:
Good one !! Thanks !!